.drone.yml

```diff
@@ -59,27 +59,42 @@ trigger:
 workspace:
   path: /tmp/teleport-plugins/test-darwin
 
+environment:
+  GOLANG_VERSION: go1.17.9
```

Comment:
This is called `GO_VERSION` in most/all the plugin Makefiles.  It'd be nice to be consistent with that for ripgrep.

Reply:
Definitely. Changing.